packages/connector-local/src/connector.ts

```diff
@@ -103,7 +103,7 @@ export default class LocalConnector implements IConnector {
     }
 
     private async fetchData(target: string, options?: IFetchOptions): Promise<FetchEnd> {
-        const content: NetworkData = await this.fetchContent(target, null, options);
+        const content: NetworkData = await this.fetchContent(target, void 0, options);
```

Comment:
Is there a reason why `void 0` is used?

Reply:
The current types required this to be `undefined` as opposed to `null`. Per our ESLint rules we shouldn't use `undefined` directly since it can be overridden, so `void 0` is the recommended approach in this case. 

Alternatively we could plumb through support for `null` instead if you'd prefer.

Reply:
> shouldn't use undefined directly since it can be overridden

Not as of ES5. 

> Alternatively we could plumb through support for null instead if you'd prefer.

@webhintio/core Thoughts?

Reply:
We could disable the `undefined` rule. I think we discussed this previously.

I don't have a strong opinion about `null` or not.

Reply:
> We could disable the `undefined` rule

@alrra does this sound good to you? If so I'll make the change to `.eslintrc.json` in this PR and switch my `void 0` usages to `undefined` instead.

Reply:
Yes, go for it.